src/components/LayoutWithRouting.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import styled from '@emotion/styled'
 import { RouteComponentProps, Router, navigate } from '@reach/router'
-import { ErrorBoundary } from 'react-error-boundary'
+import * as Sentry from '@sentry/react'
```

Comment:
Why not `import { ErrorBoundary } from '@sentry/react'` instead? That's the syntax we use everywhere